examples/Hsc.yaml

```diff
@@ -2,4 +2,5 @@
 # build the list of data IDs to analyze
 brightSnr: 100  # Only consider stars with higher S/N than this limit
 visits: [903332, 903340, 903982, 904006, 904350, 904378, 904828, 904846]
-filter: ['HSC-R', 'HSC-R', 'HSC-I', 'HSC-I', 'HSC-I', 'HSC-Y', 'HSC-Y', 'HSC-I', 'HSC-I']
+filter: ['HSC-R', 'HSC-R', 'HSC-I', 'HSC-I', 'HSC-Y', 'HSC-Y', 'HSC-I', 'HSC-I']
+ccd: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103]
```

Comment:
Is this listing all of the CCDs or *almost* all of the CCDs.  If all, could you explain why it's necessary to specify?  If *almost* all could you mention which ones are missing?

Reply:
This is actually necessary for the HSC scripts to use `makeRunList.py`.  I'm open to suggestions for how to specify "Do all CCDs" to `makeRunList.py`, but it wasn't immediately obvious to me.

As to almost all vs. all, I just used the list from the original `runHscTest.sh`, which was `0..103`.  [This](https://www.subarutelescope.org/Observing/Instruments/HSC/CCDPosition_20170212.png) makes me think that is the right way to go, since those CCDs numbered higher than 103 are focus/guider sensors.